geozero/src/mvt/mvt_error.rs

```diff
@@ -0,0 +1,18 @@
+//! MVT error type.
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum MvtError {
+    #[error("invalid feature.tags length: {0}")]
+    InvalidFeatureTagsLength(usize),
+    #[error("invalid key index {0}")]
+    InvalidKeyIndex(u32),
+    #[error("invalid value index {0}")]
+    InvalidValueIndex(u32),
+    #[error("unsupported value type for key {0}")]
+    UnsupportedKeyValueType(String),
+    #[error("geometry format")]
+    GeometryFormat,
+    #[error("Too few coordinates in line or ring")]
```

Comment:
Probably should keep error formatting consistent with capitalization. I'm ok to keep it all lower or all Title - whatever makes sense?

```suggestion
    #[error("too few coordinates in line or ring")]
```